docs/whats-new.asciidoc

```diff
@@ -13,194 +13,123 @@ Other versions: {security-guide-all}/8.6/whats-new.html[8.6] | {security-guide-a
 
 *Prebuilt rule updates*
 
-Check out the {security-guide}/prebuilt-rule-8-7-1-prebuilt-rules-8-7-1-summary.html[latest updates] to prebuilt rules. To download the latest updates, refer to {security-guide}/rules-ui-management.html#download-prebuilt-rules[Download latest Elastic prebuilt rules]. 
+Check out the {security-guide}/prebuilt-rules-downloadable-updates.html[latest updates] to prebuilt rules. To download the latest updates, refer to {security-guide}/rules-ui-management.html#download-prebuilt-rules[Download latest Elastic prebuilt rules]. 
 
 [float]
-== New integrations 
+== Alerts enhancements 
 
-The following security integrations have been added in 8.7: 
+*Control alert notifications and summaries*
 
-* AuditD
-* Content Delivery Network
-* Credential Management
-* Database
-* EDR/XDR
-* Email Security
-* Firewall
-* Identity & Access Management
-* Threat Intelligence
-* Vulnerability Management
-
-The Integrations page also has improved categorization and search functionality.
+The following enhancements give more control over how and when alert notifications are sent. For more information, refer to {security-guide}/rules-ui-create.html#rule-notifications[Set up alert notifications]. 
 
+* You can now specify how often {security-guide}/rules-ui-create.html#rule-notifications[alert notifications] are sent to third-party systems (such as Slack, JIRA, email, etc.). You can apply your preferred frequency to all rule actions, or set notification frequency individually for each action. 
++
 [role="screenshot"]
-image::whats-new/images/8.7/integrations.png[Integrations page]
+image::whats-new/images/8.8/action-frequency.png[Rule action frequency]
++
+* You can decide whether to be notified each time an alert is generated, or receive alert summaries. 
 
+* Instead of turning rules off to stop alert notifications, you can {security-guide}/rules-ui-management.html#snooze-rule-actions[snooze rule actions] for a specified time period. When you snooze rule actions, the rule continues to run on its defined schedule, but won’t perform any actions or send alert notifications. 
 
-[float]
-== Role-based access control (RBAC) for response actions
+*Max alerts warning*
 
-New Endpoint management feature privileges in {kib} allow admins to have more granular control over {security-guide}/endpoint-management-req.html[{elastic-defend} features]. Role-based access can now be managed for the following:
+When a rule reaches the maximum number of alerts it can generate in a single rule execution, the following {security-guide}/alerts-ui-monitor.html#troubleshoot-max-alerts[warning] is displayed on the rule’s details page and in the rule execution log: `This rule reached the maximum alert limit for the rule execution. Some alerts were not created.` To troubleshoot this event, we recommend you investigate alerts for any anomalies. For more information, refer to {security-guide}/alerts-ui-manage.html[Manage detection alerts].
```

Comment:
```suggestion
When a rule reaches the maximum number of alerts it can generate in a single rule execution, the following {security-guide}/alerts-ui-monitor.html#troubleshoot-max-alerts[warning] is displayed on the rule’s details page and in the rule execution log: `This rule reached the maximum alert limit for the rule execution. Some alerts were not created.` To troubleshoot this event, we recommend you check for unexpected alerts. For more information, refer to {security-guide}/alerts-ui-monitor.html#troubleshoot-max-alerts[Troubleshoot maximum alerts warning].
```